cirq/contrib/qasm_import/_parser.py

```diff
@@ -22,6 +23,10 @@
 from cirq.contrib.qasm_import._lexer import QasmLexer
 from cirq.contrib.qasm_import.exception import QasmException
 
+if TYPE_CHECKING:
+    # pylint: disable=unused-import
+    from typing import Tuple
```

Comment:
This is for the line

`for qubits in op_qubits:  # type: Tuple[cirq.Qid]`

?

That refers to the type of the loop variable? You should instead change the `type: np.broadcast` on the previous line. For example:

`op_qubits = cast(Sequence[Sequence[ops.Qid]], functools.reduce(np.broadcast, args))`

Reply:
I didn't even look at how `Tuple` was used. I'm just trying to shut pylint up...

Reply:
Done

Reply:
I'm wondering why pylint is reporting this as an error for you and not on Travis / or on my machine. 
Is it the right version? `dev_tools/conf/pip-list-dev-tools.txt` declares `pylint==2.3.*`

Reply:
I have pylint 2.3.1